src/content/staking/solo/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Solo stake your ETH
+description: An overview of how to get started solo staking your ETH
+lang: en
+template: staking
+emoji: ":money_with_wings:"
+sidebar: true
+image: ../../../assets/staking/leslie-solo.png
+alt: Leslie the rhino on her own computer chip.
+sidebarDepth: 2
+summaryPoints:
```

Comment:
I feel like a primary point should be earning the most rewards - unlike every other option, no one takes a cut of your protocol rewards.

Reply:
Perhaps we rework or replace the final point?

> Stakers receive rewards directly from the protocol for keeping their validator properly functioning and online.

I think max rewards is a more powerful value-add, though perhaps worth mentioning that rewards are direct as well?